src/plugin.js

```diff
@@ -523,15 +522,16 @@ class VR extends Plugin {
           // Native WebVR Head Mounted Displays (HMDs) like the HTC Vive
           // also need the cardboard button to enter fully immersive mode
           // so, we want to add the button if we're not polyfilled.
+
+          this.controls3d = new VRControls(this.camera);
           if (!this.vrDisplay.isPolyfilled) {
             this.addCardboardButton_();
           }
-        // FIREFOX doesn't report the polyfill display as a
-        // VRDisplay so this would error even though the video would
-        // work
-        } else if (!videojs.browser.IS_FIREFOX) {
-          this.triggerError_({code: 'web-vr-no-devices-found', dismiss: false});
+        } else {
+          this.controls3d = new OrbitControls(this.camera, this.renderedCanvas);
         }
+
+        this.requestAnimationFrame(this.animate_);
```

Comment:
I assume this is because we don't want to animate if we're in `web-vr-out-of-date` or `web-vr-not-supported` states? and the rAF is so that this method will finish up before calling animate?